src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertiesArray = sourceString.split(';');
+
+  return propertiesArray.reduce((style, current) => {
+    const [key, value] = current.split(':');
+    const styleKey = key
```

Comment:
You can write it without ternary operator `const styleKey = key.trim() || null;`

Reply:
key.trim() doesn`t work properly without nullish operator. 

const styleKey = key?.trim() || null; will be work properly, but our version of lenter doesn`t support nullish ?.

Without ternary I will be have error:  Cannot read property 'trim' of undefined